.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -640,7 +591,7 @@ stages:
 
 - stage: integration_tests_windows_iis
   condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'))
-  dependsOn: [build_windows_tracer, package_windows, generate_variables, master_commit_id]
+  dependsOn: [build_and_package_windows, build_and_package_windows, generate_variables, master_commit_id]
```

Comment:
```suggestion
  dependsOn: [build_and_package_windows, generate_variables, master_commit_id]
```

Reply:
Indeed. Fixed in 5edb6ffac1b5d96e5201336a81a5e56193263dd1